app/controllers/annual_leave_requests_controller.rb

```diff
@@ -23,13 +23,13 @@ def create
   def confirm; end
 
   def approve
-    line_reports_leave_requests = AnnualLeaveRequest.where(user: current_user.line_reports)
-    @annual_leave_request = line_reports_leave_requests.find(params[:annual_leave_request_id])
+    @annual_leave_request = AnnualLeaveRequest.find(params[:annual_leave_request_id])
+    redirect_to root_path if current_user != @annual_leave_request.user.line_manager
   end
 
   def update_status
-    line_reports_leave_requests = AnnualLeaveRequest.where(user: current_user.line_reports)
-    @annual_leave_request = line_reports_leave_requests.find(params[:annual_leave_request_id])
+    @annual_leave_request = AnnualLeaveRequest.find(params[:annual_leave_request_id])
+    redirect_to root_path if current_user != @annual_leave_request.user.line_manager
```

Comment:
@ryanb-gds is this what you meant? For some reason Brakeman is still flagging this as a security issue (see picture) 
![image](https://github.com/jyoung-gds/govuk-holiday-logger/assets/94846816/fb112fc0-6a68-4c90-bc1b-c21557292375)

Reply:
Yeah, that's really interesting. Not sure I agree with Brakeman to be honest, but in the context of a Rails application where controller variables can easily turn up in templates by accident maybe that's sensible. I guess leave it as is!